docs/current/data-model-howto.md

```diff
@@ -182,24 +184,61 @@ Boots exist as individual documents under the `boot` subdirectory. They are refe
     └── etcd-proxy.yml
 ```
 
+
 ### Sites
 
 Sites are represented by individual subdirectories. One directory == one site == one managed group of machines. Each SoT can contain multiple sites. Each of these sites shares the same assets/boot/os configuration files.
 
 Each site has _at least_ two documents, the specially named `env.yml` and at least one document describing an inventory of hosts. You may use YAML's triple-dash `---` separator to combine multiple inventory documents into one file.
 
+#### Clusters
+
+Clusters are represented by individual yaml files under a site (conventional they are prefixed with `cluster-` in the file name (example: `cluster-a.yml`).
```

Comment:
Just a small nit because I get held up on 'conventional', consider this wording:
"Clusters are defined by individual files in a site directory, prefixed with `cluster-` by convention (example: `cluster-dev.yml`)."